### PR TITLE
feat(ubiquiti): add show ipv6 route template

### PR DESCRIPTION
**Pull request by guillaume-mbali**
**ISSUE TYPE**
New Template Pull Request

**COMPONENT**
ubiquiti_edgerouter_show_ipv6_route.textfsm, edgerouter, show ip6 route

**SUMMARY**
Added the ubiquiti edgerouter template to show ipv6 route

### CODE REVIEW

File: ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Value Filldown CODE (\w{1,2})
```
Comment (jmcgill298) [inline ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm:1]: The test file does not include an example where Filldown would be needed. Is this needed, and the test case needs to be updated, or is this not needed?

File: ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Value DISTANCE (\d+)
+Value METRIC (\d+)
+Value NEXTHOP_IP ([A-F0-9:]+)
+Value INTERFACE (\w+)
+Value UPTIME (\d[\w:\.]+)
+
+Start
+  ^${CODE}\s+${IP}/${PREFIX_LENGTH}\s\[${DISTANCE}/${METRIC}\]\svia\s${NEXTHOP_IP},\s${INTERFACE},\s${UPTIME}\s* -> Record
+  ^${CODE}\s+${IP}/${PREFIX_LENGTH}\svia\s${NEXTHOP_IP},\s${INTERFACE},\s${UPTIME}\s* -> Record
+  ^\s* -> Clearall
```
Comment (jmcgill298) [inline ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm:13]: Is there a scenario where this comes into play? I don't see it in the test case.

File: ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm (hunk 1 above)
Comment (guillaume-mbali) [reply ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm]: The filldown is indeed not necessary. I have removed it.

File: ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm (hunk 2 above)
Comment (guillaume-mbali) [reply ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm]: I have made some changes to better fit with the test cases.

File: ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Start
+  ^.*IP\sRoute\sTable -> IP
+
+IP
+  ^${CODE}\s+${IP}/${PREFIX_LENGTH}\s\[${DISTANCE}/${METRIC}\]\svia\s${NEXTHOP_IP},\s${INTERFACE},\s${UPTIME}\s* -> Record
+  ^${CODE}\s+${IP}/${PREFIX_LENGTH}\svia\s${NEXTHOP_IP},\s${INTERFACE},\s${UPTIME}\s* -> Record
+  ^\s*$$
+  ^. -> Error
+
+EOF
```
Comment (jmcgill298) [inline ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm:19]:
```suggestion
```

Without a `Filldown`, the default EOF behavior should work

**Review by jmcgill298 — CHANGES_REQUESTED** [review]

File: ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm (hunk 3 above)
Comment (guillaume-mbali) [reply ntc_templates/templates/ubiquiti_edgerouter_show_ipv6_route.textfsm]: It's corrected, thank you.

**Review by jmcgill298 — APPROVED** [review]

**Comment by jmcgill298** [comment]
thanks @guillaume-mbali

**Pull request merged**